spring-web/src/main/java/org/springframework/web/service/invoker/RequestBodyArgumentResolver.java

```diff
@@ -68,9 +71,16 @@ public boolean resolve(
 			return false;
 		}
 
+		parameter = parameter.nestedIfOptional();
```

Comment:
I don't think we need this here since the only place we check the parameter type is in the reactive section, but we don't expect an `Optional` wrapper around a reactive/async type.